### PR TITLE
MSTest.Sdk: document EnablePlaywright and EnableAspireTesting

### LINKED ISSUE #2819: Add documentation for new features of MSTest.Sdk
Document Aspire and Playwright features along with default using.

AB#2051794

### PR DESCRIPTION
**Pull request by Evangelink**
## Summary

Fixes https://github.com/microsoft/testfx/issues/2819

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-sdk.md](https://github.com/dotnet/docs/blob/b003e7a98e0961fe036974c6f1497dd8a4e5e61b/docs/core/testing/unit-testing-mstest-sdk.md) | [MSTest SDK overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-sdk?branch=pr-en-us-40774) |

### CODE REVIEW

File: docs/core/testing/unit-testing-mstest-sdk.md [hunk 1]
```diff
@@ -140,6 +140,54 @@ Or to disable an extension that is coming from the selected profile. In this cas
 </Project>
 ```
 
+## Features
+
+Outside of the selection of the runner and runner specific extensions, `MSTest.Sdk` also provides additional features to simplify and enhance your testing experience.
+
+### Aspire
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-sdk.md:147]:
```suggestion
### .NET Aspire
```

File: docs/core/testing/unit-testing-mstest-sdk.md [hunk 2]
```diff
@@ -140,6 +140,54 @@ Or to disable an extension that is coming from the selected profile. In this cas
...
+
+`.NET Aspire` is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information about it, check out its [documentation page](https://learn.microsoft.com/dotnet/aspire/get-started/aspire-overview).
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-sdk.md:149]:
```suggestion
.NET Aspire is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information, see the [.NET Aspire docs](/dotnet/aspire/get-started/aspire-overview).
```

File: docs/core/testing/unit-testing-mstest-sdk.md [hunk 3]
```diff
@@ -140,6 +140,54 @@ Or to disable an extension that is coming from the selected profile. In this cas
...
+    </PropertyGroup>
+
+    <!-- references to the code to test -->
+
+</Project>
+```
+
+### Playwright
+
+`Playwright` enables reliable end-to-end testing for modern web apps. For more information about it, check out its [documentation page](https://playwright.dev/dotnet/docs/intro).
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-sdk.md:171]:
```suggestion
Playwright enables reliable end-to-end testing for modern web apps. For more information, see the official [Playwright docs](https://playwright.dev/dotnet/docs/intro).
```

**Review by IEvangelist — APPROVED** [review]

File: docs/core/testing/unit-testing-mstest-sdk.md (hunk 2 above)
Comment (Evangelink) [reply docs/core/testing/unit-testing-mstest-sdk.md]: Oooooh! I was looking on disk and I couldn't find this path that's why I used the full URL.

**Comment by Evangelink** [comment]
As usual thanks for the prompt review @IEvangelist

**Pull request merged**